docs/cmdlet-example/command-line-simple-example.md

```diff
@@ -0,0 +1,211 @@
+# Creating a cross-platform binary module with the .NET Core command-line interface tools
+
+This example uses the [.NET Core command-line interface tools][dotnet-cli] (aka
+`dotnet CLI`) to demonstrate how to create a binary module that is portable across operating
+systems supported by `PowerShell Core` as well as `Windows PowerShell` version 3 and higher.
+
+Because the binary module's assembly will be created as a `.NET Standard 2.0 class library`,
+the same assembly can be imported into both PowerShell Core and Windows PowerShell.
+This means you do not have to build separate assemblies to target these two different
```

Comment:
Going to reword to `build and distribute separate assemblies that target these two
different implementations of PowerShell.`